helm/ako/values.yaml

```diff
@@ -50,6 +50,7 @@ L7Settings:
 L4Settings:
   advancedL4: "false" # Use this knob to control the settings for the services API usage. Default to not using services APIs: https://github.com/kubernetes-sigs/service-apis
   defaultDomain: "" # If multiple sub-domains are configured in the cloud, use this knob to set the default sub-domain to use for L4 VSes.
+  autoFQDN: "<svc>.<ns>.<subdomain>" # ENUM: <svc>-<ns>.<subdomain>, <svc>.<ns>.<subdomain>, "false" If the value is false then the FQDN generation is disabled.
```

Comment:
Should we instead make the enums as:
```
- "default" (<svc>.<ns>.<subdomain>)
- "withNamespace" (<svc>-<ns>.<subdomain>)
- "false"
```

Reply:
yes, i think we should think around strings (all caps maybe), like the shardSize enums, SMALL/MEDIUM/LARGE
I also feel that with the annotation, this might just be adding additional complexity. 

Also, I might have not understood this properly but I see
`nil/"false" == <svc>.<ns>.<subdomain>`
So there are essentially two options: namespacedSubdomain (name.ns.com) and name-namespace separated fqdn (name-ns.com)
namespacedSubdomain is the current default so effectively this could be a bool?
If the only value add from the flag is name-namespace separated fqdn (name-ns.com) then that can actually be specified in the annotation itself?

Reply:
Done, called them 'default', 'flat' and 'disabled'

Reply:
got the difference between disabled and default thanks